core/modules/user/config/layout.layout.login.json

```diff
@@ -0,0 +1,56 @@
+{
+    "_config_name": "layout.layout.login",
+    "path": "",
+    "name": "login",
+    "title": "Login page Layout",
```

Comment:
Lets be consistent and omit the `Layout` bit:
```suggestion
    "title": "Login page",
```

We're only adding that suffix for the default layout and the default admin layout, because plain `Default` and `Default administrative` would not stand on their own; whereas we use plain `Dashboard`, and `Home page`, w/o the "layout" suffix:
<img width="188" alt="image" src="https://github.com/backdrop/backdrop/assets/2423362/05f14ab1-bab1-4f52-bd2d-abcb13db62ae">